distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1414,6 +1432,21 @@ type Table
              an `Unquoted_Delimiter`
            - If there are more than 10 issues with a single column,
              an `Additional_Warnings`.
+
+       ? Example Cross Tab Operation
+
+         Input Table `table`:
+
+            A | B       | C
+           ---|---------|---------
+            A | Name    | Example
+            A | Country | France
+
+         Result `table.cross_tab ['A'] 'B' (Aggregate_Column.First 'C')`:
+
+            A | Name    | Country
+           ---|---------|---------
+            A | Example | France
```

Comment:
Btw. this leads me to thinking.

If I want to reconstruct the table from an earlier `transpose` using `cross_tab`, I'll do sth like you did here.

Shall we have something like `Aggregate_Column.Only_Row` which will act like `First` but if there is more than 1 record in each group it would error? This way I can make sure that the reverse-transpose worked correctly and verify if there were no data corruptions along the way.

Not sure how needed that is, but I thought it might be useful.